external/vcm/vcm/catalog.yaml

```diff
@@ -679,12 +679,30 @@ sources:
       urlpath: "gs://vcm-ml-raw-flexible-retention/2021-01-04-1-year-C384-FV3GFS-simulations/plus-4K/C384-to-C48-diagnostics/pressure-interpolated.zarr"
       consolidated: True
 
+  2020_1yr_pire_pressure_interpolated_vertical_wind:
+    description: >-
+      Vertical wind 3D from 1 year PIRE simulation post-spinup, coarsened to C48 resolution and interpolated to pressure levels.
+      Separate from the rest of the 3D verification for this dataset as it was not part of the nudged run reference output and was taken from restarts.
+      Saved at 3h resolution, though about 80 timesteps out of the year are dropped from the final verification dataset as the dataflow job produced
+      corrupt data at those times.
+      Generated by https://github.com/ai2cm/explore/blob/master/annak/2022-01-05-PIRE-updated-input-data/2022-03-03-remove-corrupted-verification-3d-timesteps.ipynb
+    driver: zarr
+    metadata:
+      grid: c48
+      simulation: 1yr_pire_postspinup
+      category: 3d
+      variables:
+        - vertical_wind
+    args:
+      storage_options:
+        project: 'vcm-ml'
+        access: read_only
+      urlpath: "gs://vcm-ml-intermediate/2021-10-12-PIRE-c48-post-spinup-verification/pire-1yr-postspinup-restarts-pressure-interpolated-vertical-wind.zarr"
+      consolidated: True
 
   2020_1yr_pire_pressure_interpolated:
     description: >-
       3D dynamical core diagnostics variables from 1 year PIRE simulation post-spinup, coarsened to C48 resolution and interpolated to pressure levels.
-      Saved at 3h resolution, though about 80 timesteps out of the year are dropped from the final verification dataset as the dataflow job produced
-      corrupt data at those times.
       Generated by https://github.com/ai2cm/explore/blob/master/annak/2022-01-05-PIRE-updated-input-data/2022-03-03-remove-corrupted-verification-3d-timesteps.ipynb
```

Comment:
Could you update the link to the notebook that was used? Would be useful to document somewhere which nudged run was used for the reference state data.